RELEASE.md

```diff
@@ -1350,7 +1350,7 @@ Shashank Gupta, Shashi, shashvat, shashvatshahi1998, Siju, Siju Samuel,
 Snease-Abq, Spencer Schaber, sremedios, srinivasan.narayanamoorthy, Steve Lang,
 Steve Nesae, Sumesh Udayakumaran, Supriya Rao, Taylor Jakobson, Taylor Thornton,
 Ted Chang, ThisIsPIRI, Thomas Deegan, Thomas Hagebols, tianyapiaozi, Tim Zaman,
-tomguluson92, Tongxuan Liu, TungJerry, v1incent, Vagif, vcarpani, Vikram Tiwari,
+tomguluson92, Tongxuan Liu, TungJerry, Tyler Satchel Orden, v1incent, Vagif, vcarpani, Vikram Tiwari,
```

Comment:
This is for an older release. You might want to add the github username on the part corresponding to the next release, at the beginning of the file. Although, the names listed here are for functional changes, not typo fixes.